com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -521,22 +533,12 @@ private void createTasks() throws CompileExceptionError {
 
         // To currently know the output resources, we need to parse the main.collectionc
         // We would need to alter that to get a correct behavior (e.g. using GameProjectBuilder.findResources(this, rootNode))
-        String excludeFoldersStr = this.option("exclude-build-folder", "");
-        List<String> excludeFolders = BundleHelper.createArrayFromString(excludeFoldersStr);
-        excludeFolders.addAll(loadDefoldIgnore());
 
-        // remove initial "/" from excluded folder names
-        for(int i = 0; i < excludeFolders.size(); i++) {
-            String excludeFolder = excludeFolders.get(i);
-            if (excludeFolder.startsWith("/")) {
-                excludeFolders.set(i, excludeFolder.substring(1));
-            }
-        }
         // create tasks for inputs that are not excluded
         for (String input : sortedInputs) {
             boolean skipped = false;
-            for (String excludeFolder : excludeFolders) {
-                if (input.startsWith(excludeFolder)) {
+            for (String excludeEntry : excluedFilesAndFoldersEntries) {
+                if (input.startsWith(excludeEntry)) {
```

Comment:
Is this the exact algorithm we use in the editor?
Can we have a comment and a link to the source (from both places) so that we can keep them in sync?

Reply:
I added comments